src/GameSrc/fullscrn.c

```diff
@@ -407,6 +407,9 @@ void fullscreen_start() {
     // KLC   uiShowMouse(NULL);
 
     CaptureMouse(true);
+
+    void SetMotionCursorForMouseXY(void);
```

Comment:
Should these be `extern void` or does that not matter for these cases?

Reply:
Yes, making commits...